Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:12
+
+# Create app directory
+WORKDIR /src
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json .npmrc ./
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
```

Comment:
it might be better if docker container has it's own git repo, so commits are not executed to the repo from the outside the container